environments/echis/app-processes.yml

```diff
@@ -9,6 +9,8 @@ management_commands:
     queue_schedule_instances:
     handle_survey_actions:
     run_sms_queue:
+    run_form_websocket_feed:
```

Comment:
@emord I'm also thinking of killing this - as far as I can tell it's only used by dimagisphere

Reply:
cfd4591